composer.json

```diff
@@ -46,6 +47,7 @@
         "preferred-install": "dist"
     },
     "extra": {
+        "contao-manager-plugin": "Contao\\FaqBundle\\ContaoManager\\Plugin",
```

Comment:
I rather meant that all packages of the type `contao-bundle` should provide the plugin as a convention.